res/database/Default/user/scr0llbaer.xml

```diff
@@ -467,30 +510,121 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="9f1cacc4-aa56-4f4a-9a9f-8c21c4bd3a7d" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="56bcb1ff-8a5d-4d91-b3f7-ad6958b3855b" creator="9551">
+			<title>
+				<de>%recreational%%athletes% aus %randomcity% und anderswo</de>
+				<en>%recreational% %athletes% from %randomcity% and elsewhere</en>
+			</title>
+			<description>
+				<de>Aktuelle News und Berichterstattungen aus den Bunt-, Amateur- und Kreisligen der Nation. Fußball, Eishockey, Basketball, Football, Tennis, Tischtennis, Leichtathletik, Kirmesboxen, Kartsport und Formel 3 &amp; 4, und vieles mehr.</de>
+				<en>Latest news and coverage from the nation's hobby, amateur and county leagues. Soccer, ice hockey, basketball, football, tennis, athletics, fun fair boxing, karting and Formula 3 &amp; 4, and much more.</en>
+			</description>
+
+			<variables>
+				<recreational>
+					<de>Freizeit|Feierabend|Hobby|Amateur</de>
+					<en>Recreational|Leisure|Hobby|Amateur</en>
+				</recreational>
+				<athletes>
+					<de>athleten|kicker und co.|-Sportfreunde</de>
+					<en>Athletes|Footballers and co.|Sports Friends</en>
+				</athletes>
+				<randomcity>
+					<de>%STATIONMAP:RANDOMCITY%</de>
+					<en>%STATIONMAP:RANDOMCITY%</en>
+				</randomcity>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="64" required="0" />
+			</jobs>
+			<data flags="1" production_limit="100" production_broadcast_limit="2" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="203" subgenres="103,300" />
+			<blocks value="1" />
+			<price min="5000" max="10000" />
+			<potential min="20" max="50" />
+			<outcome min="20" max="50" />
+			<review min="20" max="50" />
+			<speed min="20" max="50" />
+			<production_time value="60" />
+			<programme_data_modifiers>
+				<modifier name="topicality::age" value="5" />
+				<modifier name="topicality::refresh" value="2" />
+			</programme_data_modifiers>
+		</scripttemplate>
+
+		<scripttemplate product="3" licence_type="1" guid="acf3207f-bfcd-4aef-9c10-a4871dcab928" creator="9551">
+			<title>
+				<de>%secondleague% Sport%report%</de>
+				<en>%secondleague% Sports %report%</en>
+			</title>
+			<description>
+				<de>Die zweite Liga kommt gleich nach der ersten! Aktuelles über alles, was sportlich zweitklassig ist im Lande. Zweite Ligen im Fußball, Eishockey, Basketball, Football, Tennis, Tischtennis. Außerdem Formel 2, Leichtathletik, Boxen, Kunstturnen, Eiskunstlauf, Reitsport und vieles mehr.</de>
```

Comment:
Same then here ... "Reitsport" etc could be come "Schaukelpferdparcours", "Brummkreiseln" ... depends on how "serious" you want to keep the format.

Reply:
Added TODO